app/templates/my-sessions/list.hbs

```diff
@@ -1,15 +1,12 @@
 <div class="row">
   <div class="sixteen column wide">
     {{#each model as |session|}}
-      {{session-card session=session}}
+      {{#if (or (eq session.status 'confirmed')(eq session.status 'accepted'))}}
```

Comment:
This is not the right approach, It is quite inefficient as well. Only accepted and confirmed sessions should be queried and displayed

Reply:
So this would be solve by serve or by frontend ?

Reply:
So would i have to do changes in model or something like that for quering only confirmed and accepted sessions?

Reply:
@maze-runnar it should be handled on server level